snap/snapcraft.yaml

```diff
@@ -15,13 +15,11 @@ apps:
     plugs: [network, home]
 
   # Mocked apps needed for testing
-  mock-service:
-    command: snap/local/bin/mock-app.sh
-    daemon: simple
-  mock-service-disabled:
+  mock-service: &ms
```

Comment:
I'm defining two app keys with identical values. Instead of replicating, I reference the first one. See https://blog.daemonl.com/2016/02/yaml.html

`&ms` defines an anchor for the mock-service object. `*ms` references it.